### ISSUE TITLE
[FEATURE]  Support WhatsApp CloudAPI channel

### ISSUE THREAD

**Issue by a5r0n** [botpress/messaging#504]
WhatsApp CloudAPI now is publicly available.
https://developers.facebook.com/blog/post/2022/05/19/whatsapp-cloud-based-api/

using the official meta service can reduce some third party costs..

**Comment by slvnperron** [comment]
@a5r0n I agree!  We plan to add more channels in the coming months including WhatsApp. In the meantime, we would welcome a community contribution!

**Comment by rsaisankalp** [comment]
Can you detail out how to add support for multiple channels

**Comment by arthurvenicio** [comment]
Have more details about cloud api support?

**Comment by ivarunseth** [comment]
Hi @slvnperron,
I have raised a PR for this feature request.
Please review.

https://github.com/botpress/messaging/pull/676
